_includes/manuals/3.0/1.2_release_notes.md

```diff
@@ -4,6 +4,12 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+#### Puppet ENC extracted to plugin
+
+The Foreman without the plugin doesn't not include any Puppetclasses, Smart Variables and no way of assigning them.
+It is still possible to assigning Environment, but only through Parameter `puppet_environment`, not in the Host form.
+For getting this funcionality Users can install `foreman_puppet` plugin.
+
```

Comment:
Add link to the discourse post for more information?

Reply:
maybe also link https://community.theforeman.org/t/puppet-in-3-0-release/24429 ?